content/docs/command-reference/exp/index.md

```diff
@@ -0,0 +1,44 @@
+# exp[eriments]
```

Comment:
Should we explain in the description that both forms are accepted? Or just call it the official one here (and in the nav) and then create another nav entry for whichever one is an alias? Like https://dvc.org/doc/command-reference/doctor

Reply:
Not sure about this. Thoughts @shcheklein ?

Reply:
we def should do something about this (title should be clean). I think the thing we did with doctor is an overkill to support and write. I would just mention somewhere in the doc that there is an alias.

but we'll need to check if `dvc exp --help` and `dvc exeriments --help` print the right link, etc

Reply:
> we'll need to check if dvc exp --help and dvc exeriments --help print the right link

They both print the same `dvc experiments` help output and link to https://man.dvc.org/experiments so I guess let's stick to the full name and just mention the short alias in the doc.